pennylane_pq/expval.py

```diff
@@ -17,13 +17,16 @@
 
 .. currentmodule:: pennylane_pq.expval
 
-In addition to the suitable default operations native to PennyLane, the devices of the ProjectQ plugin support a number of additional operations that can be used alongside the native PennyLane operations when defining quantum functions:
+In addition to the suitable default operations native to PennyLane,
+the devices of the ProjectQ plugin support a number of additional expectations
+that can be used alongside the native PennyLane expectations when defining
+quantum functions:
 
 .. autosummary::
    .. AllPauliZ
```

Comment:
Two things:
i) This file does not appear directly in the docs (it can be found by searching, but not from clicking links on other pages). Do we want to include it as a first-class member of the docs?
ii) (potentially related to first): this doc currently doesn't display anything of interest, since the only thing listed under `autosummary` is currently commented out

Reply:
That's a stub file for when we support exvals with multiple outcomes. I just didn't want to loose the code.